### PR TITLE
Add import/export layout actions to View menu

### PR DESCRIPTION
**Pull request by defunctzombie**
**User-Facing Changes**
Users can import/export the current layout using the _View_ menu.

Screenshot 2023-07-05 at 10 50 46 AM

**Description**

This change adds two new items to the _View_ menu. These items allow a user to export the current layout to a file or import a file to load as the current layout.

### CODE REVIEW

**Comment by 2metres** [comment]
I think so. The only other place I could see adding it would be under a dedicated Layout/Workspace item

**Comment by 2metres** [comment]
I would also advocate for adding a keyboard shortcut like Cmd `I` or `L`

File: packages/studio-base/src/components/AppBar/AppMenu.tsx [hunk 1]
```diff
@@ -178,9 +178,31 @@ export function AppMenu(props: AppMenuProps): JSX.Element {
           handleNestedMenuClose();
         },
       },
+      {
+        type: "divider",
+      },
+      {
+        type: "item",
+        label: rightSidebarOpen ? t("importLayoutFromFile") : t("importLayoutFromFile"),
```
Comment (foxymiles) [inline packages/studio-base/src/components/AppBar/AppMenu.tsx:186]: What is this ternary for?

**Review by foxymiles — APPROVED** [review]

File: packages/studio-base/src/components/AppBar/AppMenu.tsx (hunk 1 above)
Comment (defunctzombie) [reply packages/studio-base/src/components/AppBar/AppMenu.tsx]: A sloppy copy. Removed.

**Pull request merged**